org-formation/040-budgets/cur.yaml

```diff
@@ -0,0 +1,54 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  resourcePrefix:
+    Type: String
+Resources:
+  CostReportBucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: !Sub "${resourcePrefix}-cost-reports"
+      AccessControl: Private
+  CostReportBucketPolicy:
+    Type: "AWS::S3::BucketPolicy"
+    Properties:
+      Bucket: !Ref CostReportBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          # Grant AWS Billing access to view bucket ACLs and policies
+          - Action:
+              - 's3:GetBucketAcl'
+              - 's3:GetBucketPolicy'
+            Effect: Allow
+            Resource: !GetAtt CostReportBucket.Arn
+            Principal:
+              Service: "billingreports.amazonaws.com"
+          # Grant AWS Billing access to create cost reports
+          - Action:
+              - 's3:PutObject'
+            Effect: Allow
+            Resource: !Sub '${CostReportBucket.Arn}/${resourcePrefix}/*'
+            Principal:
+              Service: "billingreports.amazonaws.com"
+          # Grant full access to the 'organizations' account
+          - Action:
+              - 's3:*'
+            Effect: Allow
+            Resource:
+              - !GetAtt CostReportBucket.Arn
+              - !Sub '${CostReportBucket.Arn}/${resourcePrefix}/*'
+            Principal:
+              AWS: "531805629419"
```

Comment:
I would prefer to grant access based on the `aws-admins` group, but I can't figure out how to reference group membership as a principal.

Reply:
I think the account already has access to it's own files so I don't think we even need this section.